docs/workflow-controller-configmap.yaml

```diff
@@ -225,8 +225,19 @@ data:
     metricsTTL: "10m"
     # IgnoreErrors is a flag that instructs prometheus to ignore metric emission errors. Default is "false"
     ignoreErrors: false
-    # Use a self-signed cert for TLS, default false
-    secure: false
+    # Use a self-signed cert for TLS, default true
+    secure: true
+    # Which temporality to use for opentelemetry, defaults to Cumulative
+    temporality: Delta
+    # Options for configuring individual metrics
+    options:
```

Comment:
I feel like there's a better name for this, but it is evading me right now

Reply:
I've changed it to "modifiers" locally, which is the best I have for now.

Reply:
That does sound better, although also feels like it could be better 😅